### ISSUE TITLE
[a11y] User Cases should be focusable

### ISSUE THREAD

**Issue by rbillings** [mozilla/fx-private-relay-add-on#219]
Reviewing accessibility guidelines, any element that can be clicked should also be focusable. The User Cases list items should be able to be navigated through the keyboard and include interactive semantics.
